beets/ui/commands.py

```diff
@@ -465,9 +465,14 @@ def summarize_items(items, singleton):
 
     if items:
         average_bitrate = sum([item.bitrate for item in items]) / len(items)
+        if items[0].format == "FLAC":
+            sample_bits = u'{}kHz/{} bit'.format(
+                round(int(items[0].samplerate) / 1000, 1), items[0].bitdepth)
         total_duration = sum([item.length for item in items])
         total_filesize = sum([item.filesize for item in items])
         summary_parts.append(u'{0}kbps'.format(int(average_bitrate / 1000)))
+        if items[0].format == "FLAC":
+            summary_parts.append(sample_bits)
```

Comment:
Any particular reason that this needs to be done in two steps, instead of just in one `if` block here?

Reply:
You're right, I've merged them.